src/components/profile/profile.tsx

```diff
@@ -29,7 +29,9 @@ export const Profile: FC = () => {
 
   return (
     <div className={styles['wrapper']}>
-      <MainButton />
+      <div className={styles['back-to-main-wrapper']}>
```

Comment:
Why do we need the wrapper?

Reply:
For absolute positioning in profile page. Main button dont have these styles but absolute positioning needed only in this case